tfx_addons/firebase_publisher/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
```

Comment:
Let's import here the component and anything that users may need to use so that they can do:

```
import tfx_addons as tfxa

tfxa.firebase_publisher.FirebasePublisher(...)
```